### PR TITLE
Dala 2128 migrate percentages

### PR DESCRIPTION
**Pull request by SiegfriedSobkowiak**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [x] IF there are changes done to the framework data models or to a database entity class, the following steps were completed
  - [x] The main branch is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [x] It's verified that the CD run is green
  - [x] Immediately after that the feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on the dev server
- [ ] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [ ] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [ ] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server
  - [ ] All implemented Social Logins have been tested manually in the UI
- [x] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-internal-storage/src/main/kotlin/db/migration/utils/MigrationHelper.kt [hunk 1]
```diff
@@ -0,0 +1,133 @@
+package db.migration.utils
+
+import org.json.JSONObject
+
+/**
+ * This class provides functionality to help with common issues that might occure while writing a migration script.
```
Comment (SiegfriedSobkowiak) [inline dataland-internal-storage/src/main/kotlin/db/migration/utils/MigrationHelper.kt:6]: typo in occur

File: dataland-frontend/tests/component/utils/DataFormat.cy.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ describe("Unit tests for formating data", () => {
     const outputArray = sampleNumbers.map((el) => {
       return formatPercentageNumberAsString(el);
     });
-    expect(outputArray).to.deep.equal(["12 %", "65.12 %", "1200 %", "100 %", "0 %"]);
+    expect(outputArray).to.deep.equal(["0.12 %", "0.65 %", "12 %", "1 %", "0 %"]);
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply dataland-frontend/tests/component/utils/DataFormat.cy.ts:29]: Done

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts [hunk 3]
```diff
@@ -252,24 +252,27 @@ describe("Component test for ThreeLayerDataTable", () => {
       },
       ammonia: {
         decarbonisation: {
-          energyMix: 0.11,
+          energyMixInPercent: 0.11,
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts]: Done

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts [hunk 4]
```diff
@@ -252,24 +252,27 @@ describe("Component test for ThreeLayerDataTable", () => {
...
         },
         defossilisation: {},
       },
       cement: {
         energy: {
-          energyMix: 0.22,
+          energyMixInPercent: 0.22,
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts]: Done

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts [hunk 5]
```diff
@@ -252,24 +252,27 @@ describe("Component test for ThreeLayerDataTable", () => {
...
         },
         technology: {},
         material: {},
       },
     };
 
     const expectedAmmoniaDecarbonisationEnergyMix = roundNumber(
-      assertDefined(p2pData.ammonia?.decarbonisation?.energyMix) * 100,
+      assertDefined(p2pData.ammonia?.decarbonisation?.energyMixInPercent) * 100,
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts]: Done

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts [hunk 6]
```diff
@@ -252,24 +252,27 @@ describe("Component test for ThreeLayerDataTable", () => {
...
+      2,
+    );
+    const expectedCementEnergyEnergyMix = roundNumber(
+      assertDefined(p2pData.cement?.energy?.energyMixInPercent) * 100,
```
[the remark this answers is not in the document]
Comment (fsoffel) [reply dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts]: Done

**Pull request merged**